### PR TITLE
customer page -back end

### PR DESCRIPTION
**Pull request by Jamalat-shamallakh**
### Related:
related to #140

### Summary:
A couple of sentences explaining the contents of the PR

### Notes:
Optional notes

### Checklist Criteria
- [ ] **Branch is up to date with master**
- [ ] **The project runs**
- - [ ] You can download and install locally, and run the project locally
- - [ ] All existing and new features run without bugs
- - [ ] The build script runs without error
- [ ] **Code is consistent**
- - [ ] Linter is used, no error messages
- - [ ] Code is consistent with agreed team style
- - [ ] Naming of files is consistent with project (e.g. all files named with camelCase)
- [ ] **No console logs**
- - [ ] Console.log is only used for crucial system messages (e.g. which port server is on).
- - [ ] Any console.logs used for debugging have been removed.

### CODE REVIEW

File: server/controller/index.js [hunk 1]
```diff
@@ -5,11 +5,12 @@ const express = require('express');
 const { checkAuth } = require('./middleware/authentication');
 const { protectRoutes } = require('./middleware/protectRoute');
 const adminHandler = require('./accountsMangment/admin');
+const { getCustomer } = require('./accountsMangment/customer/getCustomers');
 const customerHandler = require('./accountsMangment/customer');
```
Comment (engshorouq) [inline server/controller/index.js]: there is customer folder for put all routes for the customer inside it and you need just add the index of this folder as wrote above 'customerHandler'

File: server/database/config/dbBuild.sql [hunk 2]
```diff
@@ -1,77 +0,0 @@
-BEGIN;
-
-    DROP SCHEMA public CASCADE;
```
Comment (engshorouq) [inline server/database/config/dbBuild.sql]: why delete `dbBuild.sql`. if this pull request merge this file will delete from the master

File: server/controller/accountsMangment/customer/getCustomers.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const { getCustomers } = require('../../../database/queries/customer/getCustomers');
+
+const getCustomer = (request, response) => {
+  getCustomers()
+    .then((res) => {
+      if (res.rows) { response.status(200).json(res.rows); }
```
Comment (engshorouq) [inline server/controller/accountsMangment/customer/getCustomers.js]: we agreed to send success data as `{result: the result will be send}`
then the response will send as like `response.status(200).send({result: res.rows});`

Comment (Jamalat-shamallakh) [reply server/controller/accountsMangment/customer/getCustomers.js]: ok

File: server/controller/index.js (hunk 1 above)
Comment (Jamalat-shamallakh) [reply server/controller/index.js]: is it true to be a separated file to handle routes for each part??

Comment (engshorouq) [reply server/controller/index.js]: in our project is required to make it

**Pull request merged**